src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php

```diff
@@ -187,6 +215,30 @@ public function deleteAction($id)
     }
 
     /**
+     * Delete an filter with the given id.
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function cdeleteAction(Request $request)
+    {
+        $ids = explode(',', $request->get('ids'));
+        if ($ids && count($ids) > 0) {
+            try {
+                $this->getManager()->batchDelete($ids);
```

Comment:
the existing will be deleted and the request answered with 204. I also added test cases for ids that do not exist.